### ISSUE TITLE
Classification - Evidence Key versions

### ISSUE THREAD

**Issue by davmlaw** [SACGF/variantgrid#245]
We need a way to store what version of data was used to fill in an evidence key - for instance all of the gnomAD fields need to say whether it was gnomad 2, 3.1 etc etc

RUNX1 are storing both 2 and 3, using the notes. Maybe we should allow multiple entries of the same eKey?

In VariantGrid we store ```Classification.variant``` and  ```Classification.annotation_version``` which was used to populate the data, so we can always go back and use that to fill in historical data once we know how we want to store it.

**Comment by TheMadBug** [comment]
Typically I would suggest this be put in the "explain" attribute.
EvidenceKeys have attributes of value, note, explain (and others like immutable, xrefs but they're for processing logic anyway).

When accepting imports we've talked about 3rd parties providing their own annotation_version set, but having independant imports be self-contained has been very handy so I would suggestion that solution (for imports anyway).

The only problem with "explain" is it's not user editable, so if you set the CADD version, and then the classifier goes and finds another CADD version and copies the value, they have no way to communicate that.
Not sure if that's going to be a problem, but something to keep in mind.

**Comment by davmlaw** [comment]
>> 3rd parties providing their own annotation_version 

Yeah I can't find this issue but from memory it was where you defined the versions of everything by POSTing to an API and then being given a key, and then using that for all classifications that had that version (which for most systems will be the same for 6-12 months)

**Comment by TheMadBug** [comment]
Oops, typo in my previous message... I would*n't* suggest that solution for 3rd parties.

Just from experience with hooking up other labs, maintaing their annotation versions (as communicated to us) is not something I think will happen reliably.

**Comment by davmlaw** [comment]
They should aspire to answer "where did you get this information you used to decide health outcomes" with more than a shrug of the shoulders
